x/vmbridge/types/protocol.go

```diff
@@ -73,7 +78,27 @@ func GetMintERC20Output(data []byte) (bool, error) {
 	return result[0].(bool), nil
 }
 
-func GetEVMABIConfig(data []byte) (abi.ABI, abi.Event) {
+func GetCallByWasmInput(callerAddr, calldata string) ([]byte, error) {
+	data, err := EvmABI.Pack(CallByWasmMethodName, callerAddr, calldata)
+	if err != nil {
+		return nil, err
+	}
+	return data, nil
+}
+
+func GetCallByWasmOutput(data []byte) (string, error) {
+	result, err := EvmABI.Unpack(CallByWasmMethodName, data)
+	if err != nil {
+		return err.Error(), err
+	}
+	if len(result) != 1 {
+		err := fmt.Errorf("%s method outputs must be one output", CallByWasmMethodName)
+		return err.Error(), err
+	}
+	return result[0].(string), nil
+}
+
+func GetEVMABIConfig(data []byte) (abi.ABI, abi.Event, abi.Event) {
```

Comment:
would it be better to return an error?

Reply:
this function is called by init() so we must be panic.